frontend/src/features/crawl-workflows/workflow-profile-filter.ts

```diff
@@ -344,6 +338,7 @@ export class WorkflowProfileFilter extends BtrixElement {
           const { checked, value } = e.target as SlCheckbox;
 
           this.selected.set(value, checked);
+          this.requestUpdate("selected");
```

Comment:
This and the other calls to `requestUpdate` can be removed with the `selected` state change.

```suggestion
          this.selected = new Map(this.selected.set(value, checked));
```